### PR TITLE
ec2_launch_template: Add option to define custom throughput

### LINKED ISSUE #1944: Supports EBS throughput for ec2_launch_template.py
### Summary

I'd like to have an option to specify custom throughput when creating a LaunchTemplate. Only IOPS can be specified in block_device_mapping as of now.
        
As of now, boto3 has this feature already implemented https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/create_launch_template.html

       

### Issue Type

Feature Idea

### Component Name

ec2_launch_template.py

### Additional Information

```yaml (paste below)
block_device_mappings:
    - device_name: /dev/sda1
      ebs:
        volume_size: "{{ item.disk_volume_size | default(20) }}"
        volume_type: gp3
        delete_on_termination: true
        throughput: 500
        iops: 5000
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by KamilBlaz**
##### SUMMARY

Fixes #1944 
Add throughput option to ec2_launch_template module in section block_device_mappings

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by markuman** [comment]
@KamilBlaz thank you very much for your contribution.  
Your PR needs at least a changelog fragment.  
Furthermore, can you please append the integration test in `tests/integration/targets/ec2_launch_template/`?  
otherwise looks good to me.

File: plugins/modules/ec2_launch_template.py [hunk 1]
```diff
@@ -119,6 +119,11 @@
           volume_type:
             description: The volume type
             type: str
+          throughput:
+            description: >
+              The throughput to provision for a gp3 volume, with a maximum of 1,000 MiB/s.
+              Valid Range - Minimum value of 125. Maximum value of 1000.
+            type: int
```
Comment (alinabuzachis) [inline plugins/modules/ec2_launch_template.py:126]: Please add `version_added: 8.1.0`

**Review by alinabuzachis — COMMENTED** [review]
@KamilBlaz Thank you for taking time to work on this new feature. Can you please add a changelog fragment https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment for this functionality and cover with integration/unit tests?

**Comment by abikouo** [comment]
@KamilBlaz Thanks for your contribution. Since the PR has not been updated and we are preparing the migration of this module, I have integrate the following changes into the PR #2164

**Pull request closed without merging**